packages/ui-components/src/components/layout/Dialog.vue

```diff
@@ -33,7 +33,7 @@
                 widthClasses
               ]"
               :as="isForm ? 'form' : 'div'"
-              @submit.prevent="onSubmit"
+              @submit.prevent="onSubmit || noop"
```

Comment:
@andrewwallacespeckle An undefined event handler (onSubmit is optional) breaks the app fyi